### PR TITLE
Add a user - Pages documentation

### PR DESCRIPTION
**Pull request by Ephraim-G**
## Changes proposed in this pull request:
- Adding Pages documentation

[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/ephraimg-patch-1)

## Security Considerations
[N/A]

### CODE REVIEW

**Comment by drewbo** [comment]
@Ephraim-G just on quick scan looks like this needs screenshot links (they don't resolve on preview) and a link to this page in the sidebar (should be a navigation.yml file you can add it to)

**Comment by Ephraim-G** [comment]
@drewbo this should be good now

**Comment by drewbo** [comment]
This is on me for not finding this sooner but we do already have a page detailing access + roles at https://cloud.gov/pages/documentation/access-permissions/ but it lacks screenshots and detailed instructions for how to add users and the email they'll receive.

@Ephraim-G what do you think about starting from that page and adding your content as appropriate? We could also preserve the new page and link between. For this page:
- The navigation needs to match between the permalink (`adding-users`) and the href (`add-user`) and then it should work (currently 404s)
- The images still aren't showing up, the best way to add them would be putting them in the `assets/pages/images` folder and referencing them like shown in this post: https://raw.githubusercontent.com/cloud-gov/cg-site/main/_pages/pages/documentation/getting-started-with-jekyll.md

**Comment by Ephraim-G** [comment]
@svenaas for your visibility, as of now no action needed

**Comment by svenaas** [comment]
Thanks! Mentioning #2275 to link PR and issue.

**Comment by Ephraim-G** [comment]
Fighting with formatting

File: _data/pages/navigation.yml [hunk 1]
```diff
@@ -46,6 +46,8 @@ sidenav:
         href: /pages/documentation/why-use-pages/
       - text: Access and permissions
         href: /pages/documentation/access-permissions/
+      - text: Adding a user to an Organization
+        href: /pages/documentation/add-user
```
Comment (apburnes) [inline _data/pages/navigation.yml]: Match the href with the permalink in the file

```suggestion
        href: /pages/documentation/adding-users/
```

File: _pages/pages/documentation/add-user.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+As a new user, reach out to your organization’s manager to be invited to the platform and subsequently the desired organization. 
+
+If you are having trouble locating who the organization manager is you can reach out in one of our slack channels #cg-pages or #cg-pages-public.
+
+## Adding a new user
+
+1. Navigate to the organizations tab
+2. Click edit in the lower right hand corner of the organization pane
+![Screenshot of the Pages admin UI interface where organization managers can access the edit feature of a singular organinzation](https://github.com/Ephraim-G/images/blob/a16563b4824af4c80e72d7813276ae35c425ed20/edit_organizations.png)
```
Comment (apburnes) [inline _pages/pages/documentation/add-user.md]: Update to the local images you added

```suggestion
![Screenshot of the Pages admin UI interface where organization managers can access the edit feature of a singular organinzation]({{ site.baseurl }}/assets/pages/images/edit_organizations.png)
```

File: _pages/pages/documentation/add-user.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+3. Click the plus sign under “Members” 
+![Screenshot of the Pages admin UI interface where organization managers can access the add memebers feature to send invitations to the organization ](https://github.com/Ephraim-G/images/blob/a16563b4824af4c80e72d7813276ae35c425ed20/add_user.png)
```
Comment (apburnes) [inline _pages/pages/documentation/add-user.md]:
```suggestion
![Screenshot of the Pages admin UI interface where organization managers can access the add members feature to send invitations to the organization ]({{ site.baseurl }}/assets/pages/images/add_user.png)
```

File: _pages/pages/documentation/permissions.md [hunk 4]
```diff
@@ -20,6 +20,34 @@ Users can be granted access to cloud.gov Pages in one of two ways:
...
 Pages Users within an Organization will be able to see all of the Sites in the Organization without having to manually add them. Pages Users can have one of two Roles within an Organization: `user` or `manager`. The roles are identical with the exception that Organization `manager`s have the ability to add/remove/manage other Pages Users within the Organization, including triggering invites to the Pages platform if necessary for new users. Organizations can only be created and deactivated by Pages Operators.
 
+
+## Adding a user to an organization
+
+
+1. Navigate to the organizations tab
+2. Click edit in the lower right hand corner of the organization pane
+ 
+ <img src="https://github.com/cloud-gov/cg-site/blob/ephraimg-patch-1/assets/pages/images/edit_organizations.png" width="60%" height="40%" />
```
Comment (apburnes) [inline _pages/pages/documentation/permissions.md]: Update to the site's image

```suggestion
 <img src="{{ site.baseurl }}/assets/pages/images/edit_organizations.png" width="60%" height="40%" />
```

File: _pages/pages/documentation/permissions.md [hunk 5]
```diff
@@ -20,6 +20,34 @@ Users can be granted access to cloud.gov Pages in one of two ways:
...
+
+3. Click the plus sign under “Members” 
+ 
+ <img src="https://github.com/cloud-gov/cg-site/blob/ephraimg-patch-1/assets/pages/images/add_user.png" width="60%" height="40%" />
```
Comment (apburnes) [inline _pages/pages/documentation/permissions.md]: Update to site's local version

```suggestion
 <img src="{{ site.baseurl }}/assets/pages/images/add_user.png" width="60%" height="40%" />
```

**Review by apburnes — COMMENTED** [review]
Looks good! Just added some updates to pull in the hosted images from the site

**Comment by Ephraim-G** [comment]
@apburnes Thanks for that, had to quickly remove two files that I had added to play around with so now you can pass it

**Review by apburnes — APPROVED** [review]
Awesome! We can close out https://github.com/cloud-gov/cg-site/issues/2275

**Pull request merged**